--- a/work/tools/build/src/tools/python.jam	2019-09-28 13:13:54.000000000 +0200
+++ b/work/tools/build/src/tools/python.jam	2019-09-28 13:14:43.000000000 +0200
@@ -545,6 +545,9 @@
 
         libraries ?= $(default-library-path) ;
         includes ?= $(default-include-path) ;
+        local conda-prefix = [ os.environ CONDA_PREFIX ] ;
+        includes = $(includes) $(conda-prefix)/Library/include ;
+        libraries = $(libraries) $(conda-prefix)/Library/lib ;
     }
     else
     {
