From 1a30b04ffc4b33db3a01f328722dc60585234806 Mon Sep 17 00:00:00 2001
From: Johan Euphrosine <proppy@google.com>
Date: Wed, 5 Oct 2022 18:11:31 +0900
Subject: [PATCH 3/3] disable osu

---
 gf180mcu/Makefile.in | 68 ++++----------------------------------------
 1 file changed, 5 insertions(+), 63 deletions(-)

diff --git a/gf180mcu/Makefile.in b/gf180mcu/Makefile.in
index 5074995..9141c0b 100644
--- a/gf180mcu/Makefile.in
+++ b/gf180mcu/Makefile.in
@@ -136,11 +136,9 @@ GF180MCU_IO_PATH = @GF180MCU_SOURCE_PATH@/gf180mcu_fd_io
 GF180MCU_SC_7T5V0_PATH = @GF180MCU_SOURCE_PATH@/gf180mcu_fd_sc_mcu7t5v0
 GF180MCU_SC_9T5V0_PATH = @GF180MCU_SOURCE_PATH@/gf180mcu_fd_sc_mcu9t5v0
 GF180MCU_SRAM_PATH = @GF180MCU_SOURCE_PATH@/gf180mcu_fd_ip_sram
-GF180MCU_OSU_SC_PATH = @GF180MCU_OSU_SC_PATH@
 
 # Path to GF180MCU library sources
 PDK_URL = https://github.com/google
-OSU_URL = https://github.com/stineje
 
 # Names of library repositories
 PDK_LIB_PR =       ${PDK_URL}/globalfoundries-pdk-libs-gf180mcu_fd_pr
@@ -148,7 +146,6 @@ PDK_LIB_IO =       ${PDK_URL}/globalfoundries-pdk-libs-gf180mcu_fd_io
 PDK_LIB_SC_7T5V0 = ${PDK_URL}/globalfoundries-pdk-libs-gf180mcu_fd_sc_mcu7t5v0
 PDK_LIB_SC_9T5V0 = ${PDK_URL}/globalfoundries-pdk-libs-gf180mcu_fd_sc_mcu9t5v0
 PDK_LIB_SRAM =     ${PDK_URL}/globalfoundries-pdk-ip-gf180mcu_fd_ip_sram
-PDK_LIB_OSU_SC =   ${OSU_URL}/globalfoundries-pdk-libs-gf180mcu_osu_sc
 
 # NOTE:  Install destination is the git repository of the technology platform.
 # Once updated in git, the git project can be distributed to all hosts.
@@ -413,7 +410,7 @@ reference: ${TECH}.json
 all: $(foreach var, ${VARIANTS}, all-$(var))
 
 # Handle prerequisites
-prerequisites: pr-repo io-repo sc-7t-repo sc-9t-repo sram-repo osu-sc-repo
+prerequisites: pr-repo io-repo sc-7t-repo sc-9t-repo sram-repo
 
 pr-repo:
 	if test "x${GF180MCU_PR_PATH}" != "x" ; then \
@@ -465,18 +462,8 @@ sram-repo:
 		fi ; \
 	fi
 
-osu-sc-repo:
-	if test "x${GF180MCU_OSU_SC_PATH}" != "x" ; then \
-		if test -d "${GF180MCU_OSU_SC_PATH}" ; then \
-			echo "Using existing installation of OSU 3.3V standard cell library from ${GF180MCU_OSU_SC_PATH}" ; \
-		else \
-			echo "Downloading OSU 3.3V standard cell library from ${GF180MCU_OSU_SC_PATH}" ; \
-			../scripts/download.sh ${PDK_LIB_OSU_SC} ${GF180MCU_OSU_SC_PATH} ; \
-		fi ; \
-	fi
-
 # Update prerequisites
-update: update-pr-repo update-io-repo update-sc-7t-repo update-sc-9t-repo update-sram-repo update-osu-sc-repo
+update: update-pr-repo update-io-repo update-sc-7t-repo update-sc-9t-repo update-sram-repo
 
 update-pr-repo:
 	if test "x${GF180MCU_PR_PATH}" != "x" ; then \
@@ -508,12 +495,6 @@ update-sram-repo:
 		../scripts/update.sh ${GF180MCU_SRAM_PATH} ; \
 	fi
 
-update-osu-sc-repo:
-	if test "x${GF180MCU_OSU_SC_PATH}" != "x" ; then \
-		echo "Updating GF180MCU OSU 3.3V standard cell library from ${PDK_LIB_OSU_SC}" ; \
-		../scripts/update.sh ${GF180MCU_OSU_SC_PATH} ; \
-	fi
-
 all-%: prerequisites
 	echo "Starting gf180mcu$* PDK staging on "`date` > ${GF180MCU$*}_make.log
 	${MAKE} general-$*
@@ -665,11 +646,11 @@ openlane-%: openlane/config.tcl openlane/gf180mcu_fd_sc_mcu7t5v0/config.tcl open
 		${OPENLANE_STAGING_$*}/rules.openrcx.gf180mcu$*.max.magic ;\
 	fi
 
-vendor-A: primitive-build-A digital-7t5v0-build-A digital-9t5v0-build-A io-build-A sram-build-A digital-osu-build-A
+vendor-A: primitive-build-A digital-7t5v0-build-A digital-9t5v0-build-A io-build-A sram-build-A
 
-vendor-B: primitive-build-B digital-7t5v0-build-B digital-9t5v0-build-B io-build-B sram-build-B digital-osu-build-B
+vendor-B: primitive-build-B digital-7t5v0-build-B digital-9t5v0-build-B io-build-B sram-build-B
 
-vendor-C: primitive-build-C digital-7t5v0-build-C digital-9t5v0-build-C io-build-C sram-build-C digital-osu-build-C
+vendor-C: primitive-build-C digital-7t5v0-build-C digital-9t5v0-build-C io-build-C sram-build-C
 
 primitive-build-%:
 	if test -d ${GF180MCU_PR_PATH} ; then \
@@ -701,12 +682,6 @@ sram-build-%:
 		make sram-$* ;\
 	fi
 
-digital-osu-build-%:
-	if test -d ${GF180MCU_OSU_SC_PATH} ; then \
-		echo "Building OSU 3.3V digital standard cell libraries" ;\
-		make digital-osu-$* ;\
-	fi
-
 primitive-%:
         # Install tech LEF and primitive devices from vendor files
 	${STAGE} -source ${GF180MCU_PR_PATH} \
@@ -831,39 +806,6 @@ digital-7t5v0-%:
 		-library digital gf180mcu_fd_sc_mcu7t5v0 2>&1 | \
 		tee -a ${GF180MCU$*}_make.log
 
-digital-osu-%:
-        # Install OSU 3.3V digital standard cells from vendor files
-	# NOTE: Work in progress (to be completed)
-	${STAGE} -source ${GF180MCU_OSU_SC_PATH} \
-		-target ${STAGING_PATH}/${GF180MCU$*} \
-		-techlef lib/9T/tlef/gf180mcu_*_tech.lef \
-			rename=gf180mcu_osu_sc_9T.tlef \
-		-liberty char/techfiles/gf180mcu_9T_TT_3P3_25C*.lib \
-			rename=gf180mcu_osu_sc_9T__tt_025C_3v30 \
-		-verilog lib/9T/verilog/*.v \
-			rename=gf180mcu_osu_sc_9T \
-		-lef lib/9T/lef/*.lef \
-		-gds lib/9T/gds/*.gds compile-only \
-		-spice lib/9T/ngspice/*.spice compile-only \
-		-xschem lib/9T/schematics/*.sch \
-		-library digital gf180mcu_osu_sc_9T 2>&1 | \
-		tee -a ${GF180MCU$*}_make.log
-
-	${STAGE} -source ${GF180MCU_OSU_SC_PATH} \
-		-target ${STAGING_PATH}/${GF180MCU$*} \
-		-techlef lib/12T/tlef/gf180mcu_*_tech.lef \
-			rename=gf180mcu_osu_sc_12T.tlef \
-		-liberty char/techfiles/gf180mcu_12T_TT_3P3_25C*.lib \
-			rename=gf180mcu_osu_sc_12T__tt_025C_3v30 \
-		-verilog lib/12T/verilog/*.v \
-			rename=gf180mcu_osu_sc_12T \
-		-lef lib/12T/lef/*.lef \
-		-gds lib/12T/gds/*.gds compile-only \
-		-spice lib/12T/ngspice/*.spice compile-only \
-		-xschem lib/12T/schematics/*.sch \
-		-library digital gf180mcu_osu_sc_12T 2>&1 | \
-		tee -a ${GF180MCU$*}_make.log
-
 io-%:
         # Install I/O cells from vendor files
 	# Note:  Do not use GF LEF views.  Annotate only.  LEF is being renamed
-- 
2.30.2

